.github/workflows/python-versions.yml

```diff
@@ -1,24 +1,21 @@
 name: python versions
 on:
-  schedule:
-    # Notifications will be sent to whoever touches this last
-    # You will be able to trigger this workflow manually if need be
-    # This runs at 4:00 every Saturday, where GHA should in theory be less busy.
-    - cron:  '0 4 * * 6'
+  pull_request:
+    paths:
+    - '.github/workflows/python-versions.yml'
+    - 'requirements*'
 
 jobs:
   # This workflow makes sure that Python dependencies install correctly for
-  # a) our current version b) the next version we're targetting and c) a futher along version
+  # a) our current version b) the next version we're targetting
   python-versions:
     name: set up
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 10
+    timeout-minutes: 20
     strategy:
       matrix:
-        # macosx-10.15 is Catalina
-        # macosx-11.0 is Big Sur, however, it takes long for jobs to get started
-        os: [ macos-10.15, macos-11.0, ubuntu-20.04 ]
-        python-version: [ 3.6, 3.8, 3.9 ]
+        os: [ macos-11.0, ubuntu-20.04 ]
+        python-version: [ 3.6.13, 3.8.10 ]
```

Comment:
`3.6.13` is the version within `.python-version`.
`3.8.10` is the minimum version required for Python to work on M1 chips.